### PR TITLE
Adding YAML for Service Account to Push Docker Image

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

This PR includes the following proposed change(s):

None

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

On the fly.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: openshift/applications/service-account/git-action-sa.yaml [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+      credentials are stored.
+    name: DEV_NAMESPACE
+    required: true
+  - displayName: Credentials Name
+    description: |
+      The name used for all components of the 
+      credentials.
+    name: NAME
+    required: true
+    value: github-cicd
```
Comment (TayGov) [inline openshift/applications/service-account/git-action-sa.yaml:648]: 😱 😨

**Review by TayGov — CHANGES_REQUESTED** [review]
Does the end of the file matter?

File: openshift/applications/service-account/git-action-sa.yaml (hunk 1 above)
Comment (akroon3r) [reply openshift/applications/service-account/git-action-sa.yaml:648]:
```suggestion
    value: github-cicd

```

**Comment by akroon3r** [comment]
@TayGov fixed

**Review by TayGov — APPROVED** [review]

**Pull request merged**